crates/state/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "svm-state"
+version = "0.1.0"
+edition = "2018"
+publish = false
```

Comment:
Please make the `[package]` section similar to the other crates

Reply:
Yeah, my bad